### PR TITLE
Updated websocket connection for much faster framerate

### PR DESCRIPTION
**Pull request by Dylan-Wheeler**
Made the websocket send only the oldest captured frame at a time instead of all previously captured frames, this seems to allow for much, much faster framerates (at least <~10ms per frame) without crashing the server.

### CODE REVIEW

**Comment by Dylan-Wheeler** [comment]
(also note that any of the other smaller changes to how the code looks are just from me enabling auto code formatting in my editor. Not actual changes that I intentionally made but they shouldn't affect functionality at all)

**Comment by Noah-Giustini** [comment]
Looks good on my end, images seem to be coming to the backend in a much more consistent fashion now

**Review by Noah-Giustini — COMMENTED** [review]
Looks good on my end, images seem to be coming to the backend in a much more consistent fashion now. Meets my approval, I will wait for Priyavart to submit their review for approval

**Review by PriyavartRajain — APPROVED** [review]
Works like a charm!

**Pull request merged**
